docs/ref-manual/cvl/rules.md

```diff
@@ -2,6 +2,15 @@ Rules
 =====
 
 Rules (along with {doc}`invariants`) are the main entry points for the Prover.
+A rule defines a sequence of [commands](statements) that should be executed
```

Comment:
I'm not sure about using 'executed' here maybe 'simulated' or 'symbolically executed' or just 'checked'?

Reply:
changed to simulated